app/models/character_class.rb

```diff
@@ -1,3 +1,14 @@
 class CharacterClass < ApplicationRecord
   has_many :virtues #TODO define the inverse_of option for our associations in models
+  belongs_to :hero
+  has_many :states
+  def self.select_values
+    CharacterClass.all.map { |character_class| [character_class.name, character_class.id, { data: { url: data_url(character_class) }}]}
```

Comment:
Consider moving to at least `helpers` level. Also please think if it is ok to make the actual form object responsible for displaying the set of options in order to "generate itself" as a form an why?